hugegraph-api/src/main/java/com/baidu/hugegraph/core/GraphManager.java

```diff
@@ -157,6 +159,10 @@ public HugeGraph cloneGraph(String name, String newName,
     }
 
     public HugeGraph createGraph(String name, String configText) {
+        E.checkArgument(hugeConfig.get(ServerOptions.ENABLE_DYNAMIC_CREATE_DROP),
```

Comment:
can rename hugeConfig to conf, and expect 'this.xx' prefix style for member access: "this.conf"